stubs/FTS-14076678.json

```diff
@@ -0,0 +1,30 @@
+{
+    "videoLimit": null,
```

Comment:
No video assets lncluded so do we need this property? Or should we have video and podcasts on this contract too so it is similar to 'CA-00001558' but set up for rich articles

Reply:
We would need to update the assets from salesforce to include video and podcast